### PR TITLE
make footer responsive on mobile/tablet

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR 20 - 12 - 23 

## Changes

- Align elements vertically on mobile and tablet and remove paragraph on the `class= "footer__socialMedia"` for better UI

Screenshot 2023-12-20 142050

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -98,42 +98,48 @@ <h1 class="header__logoTitle">Pet Care</h1>
...
-            <img src="/Assets/images/footer/facebook_footer.svg" alt="Facebook icon" title="Get me to Facebook">
-        </a>
-        <a href="https://www.instagram.com/">
-            <img src="/Assets/images/footer/instagram_footer.svg" alt="Instagram icon" title="Get me to Instagram">
-        </a>
-        <a href="https://www.youtube.com/">
-            <img src="/Assets/images/footer/youtube_footer.svg" alt="Youtube icon" title="Get me to Youtube">
-        </a> 
+        <div><p>© 2023. Pet Care S.R.L.</p></div>
+        <div class="footer__mediaContainers">
```
Comment (vladiiancu) [inline home.html]: Not a big fan of the name. It's not semantic.

Comment (vladiiancu) [reply home.html]: @Alexandru-Bulai any input here?

Comment (vladiiancu) [reply home.html]: @Alexandru-Bulai another comment as requested

Comment (Alexandru-Bulai) [reply home.html]: > Not a big fan of the name. It's not semantic.

I changed the name to 'footer__icons,' but shortly after, I realized that I was not using that class, so I removed it completely

**Review by vladiiancu — APPROVED** [review]

**Pull request merged**